src/Languages/Editor/Test/Fakes/Shell/TestUIServices.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Microsoft.Common.Core.Test.Fakes.Shell;
+using Microsoft.Common.Core.UI;
+using Microsoft.Common.Core.UI.Commands;
+
+namespace Microsoft.Languages.Editor.Test.Fakes.Shell {
+    public class TestUIServices : IUIService {
```

Comment:
Is this somehow different from `Microsoft.Common.Core.Test.Fakes.Shell.TestUIServices`?